FraihmworkHealthAndIntegrityApi.yaml

```diff
@@ -1787,6 +2005,48 @@ components:
         - sourceId
         - description
         - timeOfBlocklist
+           
+    UnavailabilitySchedule:
+      type: object
+      required:
+        - componentId
+        - schedules
+      properties:
+        componentId:
+          type: string
+          format: uuid
+        schedules:
+          type: array
+          items:
+            type: object
+            properties:
+              days:
+                type: array
+                items:
+                  type: string
+                  enum:
+                    - Monday
+                    - Tuesday
+                    - Wednesday
+                    - Thursday
+                    - Friday
+                    - Saturday
+                    - Sunday
+                    - All
+              timePeriods:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    startTime:
+                      type: string
+                      format: time
```

Comment:
Is 'time' a recognized format by swagger? I don't see it here: https://swagger.io/docs/specification/data-models/data-types/#string

Reply:
If not, you may get more mileage out of a 'pattern' object and define this as a regular expression

Reply:
yes 'time' is not recognized. accidentally removed it probably.
i meant format: date-time

Reply:
I'm not sure date-time is appropriate either, as this would imply a 'date' portion of the string, which we do not want